content/docs/dvclive/api-reference/get_step.md

```diff
@@ -21,4 +21,8 @@ while dvclive.get_step() < 3:
 DVCLive uses the `step` to track the progress of each metric logged with
 `dvclive.log()`.
 
-Each call to `dvclive.next_step()` increases the `step` count.
+Each call to `dvclive.next_step()` increases the `step` count, making `[0, N)`
+the default range of `step` values.
+
+> 💡 You can alter the default range of `step` values by calling
+> `dvclive.set_step()`
```

Comment:
This is a bit cryptic/ mathematical. How about something like

> ... increases the `step` count by 1 (one). You can use `dvclive.set_step()` to increment the step count by any other number.

Reply:
`dvclive.set_step()` needs an auto-link entry BTW 🙂

Reply:
> dvclive.set_step() needs an auto-link entry BTW 🙂

It should be working now (it works for me in the deployment)